clerk/clerk.go

```diff
@@ -26,8 +26,8 @@ type Client interface {
 	NewRequest(method string, url string, body ...interface{}) (*http.Request, error)
 	Do(req *http.Request, v interface{}) (*http.Response, error)
 
-	DecodeToken(token string) (*Claims, error)
-	VerifyToken(token string) (*Claims, error)
+	DecodeToken(token string) (*TokenClaims, error)
+	VerifyToken(token string) (*SessionClaims, error)
```

Comment:
❓ Isn't this a breaking change? Or do these new types have the same API as the previous one? (Even so, it's still breaking for users that checked the actual type).

Reply:
After we add these function utilities, we haven't released a new sdk version so i doubt anyone are using them at all

Reply:
Yeah forgot that 😊